sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/ResourceParserTests.cs

```diff
@@ -60,7 +60,7 @@ public void ServiceNamespaceFromResource()
             var resourceParser = new ResourceParser();
             resourceParser.UpdateRoleNameAndInstance(resource);
 
-            Assert.StartsWith("my-namespace.unknown_service", resourceParser.RoleName);
+            Assert.StartsWith("[my-namespace]/unknown_service", resourceParser.RoleName);
```

Comment:
Is it acceptable to send `unknown_service` to the ingestion service as per schema?

Reply:
it is but I think it needs to be updated.